Tests/test_file_jpeg2k.py

```diff
@@ -34,7 +34,8 @@ def test_sanity(self):
         self.assertRegexpMatches(Image.core.jp2klib_version, '\d+\.\d+\.\d+$')
 
         im = Image.open('Tests/images/test-card-lossless.jp2')
-        im.load()
+        px = im.load()
+        self.assertTrue(px)
```

Comment:
Maybe a bit more strict check there? For example that the object implements getitem: `px[0,0]`.